app/views/projects/show.html.erb

```diff
@@ -185,8 +185,16 @@
       <%= render layout: "shared/box", locals: {title: 'Fields', width: '12', unique_id: 'fields', should_hide: false} do %>
         <% if (@project.fields.count == 0) && can_edit?(@project) %>
           <div class="col-md-12 center">
-            <a href="/projects/<%=@project.id%>/edit_fields"><button class="btn btn-success" id="manual_fields"><i class="fa fa-keyboard-o" style="margin:5px"></i>Setup Manually</button></a>
-            <button class="btn btn-success" id="template_file_upload"><i class="fa fa-file-o" style="margin:5px"></i>Upload File</button>
+            <a href="/projects/<%=@project.id%>/edit_fields">
+              <button class="btn btn-success" id="manual_fields">
+                <i class="fa fa-keyboard-o" style="margin:5px"></i>
```

Comment:
It's only these two.  Out of every single `<i>` tag on rSENSE, it's only this guy with `fa-keyboard-o` and `fa-file-o`.  And those classes don't even live in /rSENSE (must come from somewhere else)?  Do you *really* want another class in the CSS just to add a `margin:5px` to two icons?